### PR TITLE
Pavan

### PR DESCRIPTION
**Pull request by tpavan**
Developed the code for WebConfig

### CODE REVIEW

File: src/test/java/tests/SampleTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class SampleTest {
+
+    static{
+        System.setProperty("webdriver.chrome.driver","C:\\Users\\ptuptewar\\Repo\\testframework\\src\\drivers\\chromedriver.exe");
+    }
+    public static WebDriver driver = new ChromeDriver();
+
+    public static void main(String[] args) throws Exception {
+        driver.get("https://paytm.com/");
+        Thread.sleep(3000);
```
Comment (Gauri-hiremath) [inline src/test/java/tests/SampleTest.java:20]: Update wait time from 3000 mil to 4000

**Review by Gauri-hiremath — CHANGES_REQUESTED** [review]
Please address review comments

File: src/test/java/tests/SampleTest.java (hunk 1 above)
Comment (tpavan) [reply src/test/java/tests/SampleTest.java:20]: will update

Comment (tpavan) [reply src/test/java/tests/SampleTest.java]: changes done

**Pull request merged**